src/app/record/components/work/work.component.ts

```diff
@@ -100,21 +100,25 @@ export class WorkComponent implements OnInit {
     contributors.forEach((c) => {
       if (this.isPublicRecord) {
         if (c?.orcid?.value === this.isPublicRecord) {
```

Comment:
maybe we can simplify this 3 `ifs` into a single

`c?.orcid?.value === this.isPublicRecord || this.id`